src/main.rs

```diff
@@ -253,8 +258,40 @@ impl EventHandler for Messages {
         self.cmds.execute(cx, msg);
     }
 
+    fn message_update(
```

Comment:
These need to be updated to use the raw event handler currently in use in master.  https://github.com/rust-lang/discord-mods-bot/blob/master/src/main.rs#L228